File: 20190413_ABC124/A-Buttons.cpp
#include <iostream>
using namespace std;

int main()
{
    int NUM_PUSH = 2;
    int A, B;
    cin >> A >> B;
    int push, sum = 0;
    for (int i = 0; i < NUM_PUSH; i++) {
        if (A >= B) {
            push = A;
            A--;
        }
        else {
            push = B;
            B--;
        }
        sum += push;
    }
    cout << sum << endl;
    return 0;
}

File: 20190413_ABC124/A-Buttons.py
NUM_PUSH = 2


def main():
    A, B = map(int, input().split())
    sum = 0
    for i in range(NUM_PUSH):
        if A >= B:
            push = A
            A -= 1
        else:
            push = B
            B -= 1
        sum += push
    print(sum)


if __name__ == "__main__":
    main()
